src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.jsx

```diff
@@ -19,43 +20,26 @@ const ColorPickerItemComponent = ({
   isMultiselect,
   isSelected,
   colorSize,
-  tooltipContent
+  tooltipContent,
+  isActive,
+  colorShape
 }) => {
   const colorAsStyle = ColorUtils.getMondayColorAsStyle(color, colorStyle);
   const itemRef = useRef(null);
 
   const onMouseDown = useCallback(e => e.preventDefault(), []);
   const onClick = useCallback(() => onValueChange(color), [onValueChange, color]);
 
-  useEffect(() => {
```

Comment:
this effect was used to change the color to "selected" mode when hovering. This behavior has changed - we now add a border as a hovering feedback, instead of changing the color